package.json

```diff
@@ -118,14 +119,15 @@
     "@types/lodash": "4.14.100",
     "@types/node": "^14.18.12",
     "angular-plotly.js": "^4.0.4",
+    "axios": "^0.27.2",
     "codelyzer": "^6.0.0",
     "electron": "13.6.6",
     "electron-builder": "22.9.1",
     "electron-download-tf": "4.3.4",
     "electron-packager": "15.2.0",
-    "jasmine-core": "~3.6.0",
```

Comment:
Why are we removing `jasmine-core` here?

Reply:
The specific version caused installation problems and it's not being directly used. It's already a sub-dependency, so it'll be installed anyways.